src/Command/Redis/ZMSCORE.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/*
+ * This file is part of the Predis package.
+ *
+ * (c) Vladyslav Vildanov <vladyslav.vildanov@redis.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code
+ */
+
+declare(strict_types=1);
+
+namespace Predis\Command\Redis;
+
+use Predis\Command\Command as RedisCommand;
+
+/**
+ * @link https://redis.io/commands/zmscore/
```

Comment:
Maybe explain (add in comments) what zmscore is and does. The link is great -we do this in redis-py, and my dream is to always do this.